src/madness/world/dqueue.h

```diff
@@ -78,9 +85,18 @@ namespace madness {
         volatile int _back;    ///< Index of element at back of buffer
         DQStats stats;
 
+#ifdef MADNESS_DQ_USE_PREBUF
+	static const size_t NPREBUF=20;
+	static __thread T prebuf[NPREBUF]; // relies on this being a singleton class!!!!!!!!!!!!!!!!!!
```

Comment:
to be portable need to use `thread_local` ... https://en.cppreference.com/w/cpp/keyword/thread_local

Reply:
Okay.